recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@
 {% set sha256 = "8b7b63528c035055bff0820cf16101299bd5f23d7c9aa28f70a99b33e9de7335" %}
 
 package:
-   name: filesystem-spec
+   name: fsspec
```

Comment:
You kept the jinja var in line `2` and that can be confusing.
I recommend either using the var everywhere or, get rid of the var and write the name instead in the URL, etc.